src/inc/aduc/aduc_banned.h

```diff
@@ -0,0 +1,18 @@
+/**
+ * @file aduc_banned.h
+ * @brief Macro definitions that cause banned functions considered to be
+ * unsafe to result in a compilation error.
+ *
+ * @copyright Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+#ifndef ADUC_BANNED_H
+#define ADUC_BANNED_H
+
+#define DEFINE_BANNED_ALTERNATIVE(BANNED_FUNC, ALT_FUNC) BANNED_FUNC##_is_banned__Please_use_##ALT_FUNC##_instead
```

Comment:
nit: Function name should be "BAN_HAMMER_TO_ALT()" because it would make me laugh.

Reply:
I like it but won't fix